web/src/app/components/app-board-form/app-board-form.component.ts

```diff
@@ -0,0 +1,46 @@
+import { BoardItem } from './../../../common/models/board';
+import { Store } from '@ngrx/store';
+import { Component, Input } from '@angular/core';
+import { AppState } from 'src/store/reducers';
+
+// actions
+import { addBoard } from './../../../store/actions';
+
+export interface AppBoardComponentInterface {
```

Comment:
it has move to separate `interface`

Reply:
latest component pushed as the requested format.